src/codegen.cpp

```diff
@@ -979,7 +979,7 @@ static Buf *panic_msg_buf(PanicMsgId msg_id) {
         case PanicMsgIdSliceWidenRemainder:
             return buf_create_from_str("slice widening size mismatch");
         case PanicMsgIdUnwrapOptionalFail:
-            return buf_create_from_str("attempt to unwrap null");
+            return buf_create_from_str("attempt to use null pointer");
```

Comment:
Unwrapping null is not specific to pointers.
```zig
var a: ?u32 = null;
_ = a.?; // 'attempt to use null pointer' ???
```

Reply:
True